core/domain/caching_services_test.py

```diff
@@ -50,6 +50,7 @@ class CachingServicesUnitTests(test_utils.GenericTestBase):
         'states_schema_version': 45,
         'param_specs': {},
         'param_changes': [],
+        'proto_size_in_bytes': 0,
```

Comment:
Is this usually going to be 0? If it's typically nonzero let's use a nonzero value in the test data. (Ditto elsewhere.)

Reply:
(Ah sorry, you have mentioned this as a TODO in the PR description.)